.github/workflows/windows.yml

```diff
@@ -59,7 +59,6 @@ jobs:
       run: tox -e py27-coverage
       env:
         MAGIC_FOLDER_HYPOTHESIS_PROFILE: "magic-folder-ci"
-        PIP_FIND_LINKS: ./wheelhouse
```

Comment:
> Is this incidental cleanup? I don't see anything that ever populated a ./wheelhouse so I guess this had no effect.

Yes. While working on this, I was looking at the output of the install, and the repeated errors from this made it more difficult to see what the actual failures were.

> And there's another ./wheelhouse reference in this config that's being left.

I guess you are looking at the merged result? I think the other instance was added in https://github.com/LeastAuthority/magic-folder/pull/483 which was after I last updated here. I'll rebase and remove it.

>  It would be easier to know if this weren't mixed in with a lot of other packaging/dependency management changes.

I've definitely struggled somewhat figuring out the right size to make changes, particularly given the sometimes long review turnaround.